.github/workflows/dev-build.yml

```diff
@@ -74,18 +74,14 @@ jobs:
           zip $NAME target/release/kdf -j
           mv $NAME ./$BRANCH_NAME/
 
-      - name: Upload output
+      - name: Upload build artifact
         env:
-          AVAILABLE: ${{ secrets.FILE_SERVER_KEY }}
-        if: ${{ env.AVAILABLE != '' }}
-        uses: easingthemes/ssh-deploy@v5.0.3
-        with:
-          REMOTE_HOST: ${{ secrets.FILE_SERVER_HOST }}
-          REMOTE_USER: ${{ secrets.FILE_SERVER_USERNAME }}
-          REMOTE_PORT: ${{ secrets.FILE_SERVER_PORT }}
-          SSH_PRIVATE_KEY: ${{ secrets.FILE_SERVER_KEY }}
-          SOURCE: "${{ env.BRANCH_NAME }}/"
-          TARGET: "/uploads/${{ env.BRANCH_NAME }}"
+          FILE_SERVER_HOST: ${{ secrets.FILE_SERVER_HOST }}
+          FILE_SERVER_USERNAME: ${{ secrets.FILE_SERVER_USERNAME }}
+          FILE_SERVER_PORT: ${{ secrets.FILE_SERVER_PORT }}
+          FILE_SERVER_KEY: ${{ secrets.FILE_SERVER_KEY }}
+        if: ${{ env.FILE_SERVER_KEY != '' }}
```

Comment:
nit: would it look prettier if we abstract this away in the python file instead?

Reply:
We should not run this step if `FILE_SERVER_KEY` is missing. Otherwise it will seem like it ran on the CI pipeline, but the script will do nothing. IMO this makes it quite magical/complicated (especially if the source is downstream).

Reply:
Agreed!